jobbergate-cli/jobbergate_cli/render.py

```diff
@@ -197,7 +197,7 @@ def render_single_result(
         result = cast(Dict[str, Any], result_dict)
 
     if ctx.raw_output:
-        print_json(json.dumps(result))
+            print_json(json.dumps(result))
```

Comment:
this might lead to errors, won't it?

Reply:
No. Python doesn't care about the number of indentations in a block. This just makes it consistent with the rest of the code.

Reply:
Even though it works, I don't think this is a good practice. The rest of the code is indented with the correct amount of spaces, why not keep it that way?

Reply:
Oh, I was looking at the diff wrong. I don't know why the extra spaces got added, but they were removed after I ran the formatters and pushed the latest commit. Sorry for the misunderstanding.

Yes, we should not have the extra indents. This change was bad.